_docs/style/spacing.md

```diff
@@ -3,59 +3,83 @@ title: Spacing
+{% endfigure %}
 
 {% endcolumn %}
+
 </div>
+
 </section>
 
+<section class="static-section" markdown="1">
+
+<div class="hxRow" markdown="1">
+
+{% column left:"hxCol hxSpan-12-xs hxSpan-12-sm hxSpan-12-md hxSpan-12-lg" %}
+
+## Best practices
+
+Consider the following best practices when you apply the Helix spacing system:
+
+- In most cases, use medium spacing (20px) as the default.
+- When choosing spacing values for a new design, consider the user's needs and use case. Use small spacing when users prefer greater data density. Use large spacing to increase whitespace, which can help enhance user comprehension.
+- Avoid uneven spacing (padding) on containers. In the majority of cases, containers must use square Inset spacing, which means that the top, right, bottom, and left spacing values are equal inside the container.
+
+{% endcolumn %}
+
+</div>
+
+</section>
 
 <section class="static-section" markdown="1">
 
-## Container
 <div class="hxRow" markdown="1">
 
 {% column left:"hxCol hxSpan-12-xs hxSpan-12-sm hxSpan-12-md hxSpan-12-lg" %}
 
-Pages, cards, modals, panes, etc. are containers that frame content within a UI. Containers relate directly to the canvas / grid to create the layout of any page. For this reason, we defined a 20px-square inset inside all containers to be harmonious with our column gutters.
+## Container spacing
+
+Pages, cards, modals, panels, and choice tiles are examples of containers that frame content within a UI. By default, apply a square inset of 20px to the top, right, bottom, and left side of the container. The 20px is the medium spacing value and you can use smaller or larger spacing, if required.
 
-{% figure [caption: "Container spacing of twenty pixels (square-inset)."][class:"image bg-light border"] %}
-<embed src="{{site.url}}/assets/images/style/spacing/spacing-system-rule-for-containers.png" width="893"/>
+{% figure [caption: "Helix containers"] [class:"image bg-light border"] %}
+<embed src="{{site.url}}/assets/images/style/spacing/spacing-container-types.png" width="635"/>
 {% endfigure %}
 
 {% endcolumn %}
+
 </div>
 
+</section>
+
+<section class="static-section" markdown="1">
+
 <div class="hxRow"  markdown="1">
 
 {% column left:"hxCol hxSpan-12-xs hxSpan-12-sm hxSpan-4-md hxSpan-4-lg" %}
 
-### Container Spacing Variants
+### Container spacing variants
 
-Some cases may require different spacing to accommodate very small or large containers (see [choice-tiles]({{site.url}}/components/choice-tiles.html)). Therefore we have included Small, Medium, and Large container spacing sizes to facilitate your content.
+In some cases you may be required to use smaller or larger spacing than the 20px default value. Use the following spacing values to apply extra-small through extra-extra-large spacing. Keep in mind that large spacing can result in too much whitespace while small spacing can lead to a data dense UI, which can be difficult for a user to process.
```

Comment:
we only have small, medium and large Option Tiles now.
Small: 12px
Medium: 20px
Large: 32px

Reply:
also, in the visual we don't need the redlines.

Reply:
Done and done!